contrib/admin/rm-whitespace.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+
+EXTENSIONS_TO_PROCESS = {'.csv',
+                         '.cwl',
+                         '.input',
+                         '.json',
+                         '.md',
+                         '.mtail',
+                         '.py',
+                         '.rst',
+                         '.sh',
+                         '.test-pr',
+                         '.tsv',
+                         '.txt',
+                         '.wdl',
+                         '.yaml',
+                         '.yml',
+                         'Dockerfile',
+                         'Makefile'}
+
+
+def strip_whitespace_from_file(filename: str):
```

Comment:
Needs docstring, type hints; should be added to the list of files to typecheck in the `Makefile`

Reply:
Should be fixed.